.github/workflows/build.yml

```diff
@@ -134,6 +134,9 @@ jobs:
       - name: Setup Database
         run: bundle exec rake db:setup
 
+      - name: Install chromedriver
+        run: apk add chromium chromium-chromedriver
```

Comment:
Need to update the README to mention the new dependency on chromedriver to run the specs